tests/tracer/test_single_span_sampling_rules.py

```diff
@@ -64,7 +65,7 @@ def test_sampling_rule_init_via_env():
 
     # Testing error thrown when neither name nor service is set
     with override_env(dict(DD_SPAN_SAMPLING_RULES='[{"sample_rate":1.0}]')):
-        with pytest.raises(ValueError):
+        with pytest.raises(ValidationError):
```

Comment:
are these errors descriptive/helpful?

do we want to catch and re-raise with a better error message?

Reply:
Hmm only if you can read jsonschemas. I'll re-raise